CHANGELOG.md

```diff
@@ -1,12 +1,22 @@
 # Changelog
 
+## v0.68.0 - 2022-06-21
+
```

Comment:
```suggestion

This release introduces a bug fix in the garbage collector.
If you used the garbage collection client over a cloud storage that is not S3 (which you shouldn't have done), you'll have to reconfigure it.
If you used the garbage collection client over S3, it will work the same way.

```